robottelo/entities.py

```diff
@@ -2288,25 +2288,60 @@ class Meta(object):
         server_modes = ('sat', 'sam')
 
 
-class SyncPlan(orm.Entity):
+class SyncPlan(orm.Entity, orm.EntityReadMixin, orm.EntityDeleteMixin,
+        orm.EntityCreateMixin):
     """A representation of a Sync Plan entity."""
-    organization = entity_fields.OneToOneField('Organization', required=True)
+    enabled = entity_fields.BooleanField(required=True)
     name = entity_fields.StringField(required=True)
     # how often synchronization should run must be one of: none, hourly, daily,
     # weekly.
     interval = entity_fields.StringField(
-        choices=('none', 'hourly', 'daily', 'weekly'),
+        choices=('hourly', 'daily', 'weekly'),
         required=True,
     )
     # start datetime of synchronization
     sync_date = entity_fields.DateTimeField(required=True)
     description = entity_fields.StringField()
 
-    class Meta(object):
```

Comment:
Have realized that the API path is being se in the `__init__` method, now makes more sense. Thanks for the clarification.